### ISSUE TITLE
jukit_comment_mark seems not work

### ISSUE THREAD

**Issue by Mickychen00** [luk400/vim-jukit#40]
Hi, I want to set the cell title with prefix "#%% ", such as "#%% Example title". So I set jukit_comment_oark variable in a jukit_conf.vim and source it in init.vim:
let g:jukit_comment_mark = '#%%'  or 
let g:jukit_comment_mark = '#%% ' (with a space)
The above two settings are both not work.
Is there any wrong with me?

**Comment by luk400** [comment]
Works as expected for me, if you visually select a cell marker you should see that the cell marker now starts with "#%%" instead of only "#" after creating a new cell.

If you mean that you don't want to highlight the cell markers using signs and just want to display them as "#%%", then that's not possible, because the cell markers also contain the cell id's which are needed to assign saved outputs to specific cells.

The variable `g:jukit_comment_mark` really only exists in case you don't use this plugin with python and want to specify a different comment character other than "#" for commenting out the cell markers (e.g. "--" in lua)

**Comment by Mickychen00** [comment]
Hi, maybe I understand your explanation. In my work, I just want to use "#%% Some title" to split the py file as several cells. And use the 
nnoremap <leader><space> :call jukit#send#section(0)<cr>
to run current cell.
Whether the command
nnoremap <leader>co :call jukit#cells#create_below(0)<cr>
is the prerequisite command before the <leader><space>.

When I use <leader><space> firstly, the whole py code will be executed, despite I set several "#%%" cells myself.

**Comment by luk400** [comment]
Yeah that does not work, you can not use "#%%" to separate cells. You can not specify your own cell markers. You need to use the [designated cell functions to create/split cells](https://github.com/luk400/vim-jukit#cells)

**Comment by DeadlySquad13** [comment]
> Yeah that does not work, you can not use "#%%" to separate cells. You can not specify your own cell markers. You need to use the [designated cell functions to create/split cells](https://github.com/luk400/vim-jukit#cells)

Hi, thanks for you plugin!
Is it possible to change cell marker to start with `# %%` or make it configurable via settings? I suppose it won't look like a useful setting at first but actually it has a lot of use-cases and your plugin will win more people.

As for me, I can't render images on my terminal so I use [jupytext](https://github.com/mwouts/jupytext) and [jupyter-ascending](https://github.com/untitled-ai/jupyter_ascending.vim). They're cool for visualization-heavy works but require web browser with jupyter notebook always opened. On the other side, your plugin allows to stay entirely in neovim when I want to play with data and calculations: any terminal is enough for that, everything is easily accessible via vim keybindings and there're also a lot of useful commands for managing cells provided by you. However, I can't combine workflow of two plugins, I get something like this when I try to use your functions:
![image](https://user-images.githubusercontent.com/46250621/209166302-ac30fa46-901d-4eed-b0f6-745665be3ea1.png)
Well, it's expected: plugins use different cell delimiter formats...

Please, consider my words as I'm sure there will be a lot folks who will try to use your plugin with another existing tools. To understand what I mean just look at [jupytext notes about format](https://github.com/mwouts/jupytext#which-text-format). It seems there're different formats out there and I think a day won't pass without someone creating their own new format : D. Though, `# %%` seems to be the most popular.

**Comment by luk400** [comment]
> I have tested and it seems to work just fine with `# %% id_or_some_other_metadata_you_need_for_your_plugin`

hi, if this works fine then it should suffice to specify `let g:jukit_comment_mark = '# %% '` in your config, because then cell delimiters will start with `# %%`, example: `# %% |%%--%%| <piKA5Zv2Wy|qWQFOOFajV>`
